modules/exploits/example.rb

```diff
@@ -0,0 +1,84 @@
+##
```

Comment:
this should have module docs, and at least have a reference to the C program that it exploits